app/models/ext_management_system.rb

```diff
@@ -177,6 +177,32 @@ def hostname_format_valid?
 
   default_value_for :enabled, true
 
+  # Sets EMS's (and children ems) enabled flag.
+  # 1. MiqServer.monitor_loop calls MiqWorker.sync_workers and runs/stops Refresh and Event workers (provider specific)
+  # 2. Disables SmartState and C&U for provider (@see ProviderSuspendMixin for methods used)
+  # - these workers are not provider specific, but provider-type specific
+  # @param value [Boolean]
+  def change_enabled_with_children!(value)
+    self.enabled = value
```

Comment:
I wonder if this is better to handle as an after hook such that after enabled= is modified, then we push down to the children.  That way the caller just calls `enabled=`, then `.save!`